app/views/layouts/_navbar.html

```diff
@@ -0,0 +1,14 @@
+<div class="navbar">
+  <a href="/">Home Page </a>
```

Comment:
One last touch, please: in Rails, we use `route helpers` in the form:
```
<%= link_to 'Home page', root_path %>
```
Try to refactor the links here to the Rails way, and you will see how easy to use they are. Hint: for example, link to the `ranklist` is: 
```
<%= link_to 'Ranklist', ranklist_path %>
```